docroot/themes/custom/mass_admin_theme/css/components/paragraphs.admin.css

```diff
@@ -76,3 +76,16 @@ div.field--widget-entity-reference-paragraphs .description > strong {
 .field--widget-entity-reference-paragraphs .form-radios label {
   word-break: break-word;
 }
+
+@media screen and (max-width: 1120px) {
```

Comment:
```suggestion
@media screen and (max-width: 1460px) {
```
I'm still seeing this issue > 1120px, I suggest increasing the max-width

<img width="1330" alt="Screen Shot 2021-11-29 at 3 45 26 PM" src="https://user-images.githubusercontent.com/5789411/143941557-3cb168a9-208e-45f5-9e8e-948e739581c0.png">